README.md

```diff
@@ -46,14 +46,23 @@ We use SASS CSS pre-compiler. If you're doing front-end work your local machine
 ### Installing
     $ git clone https://github.com/uktrade/directory-ui-export-readiness
     $ cd directory-ui-export-readiness
-    $ virtualenv .venv -p python3.5
+    $ virtualenv .venv -p python3.6
     $ source .venv/bin/activate
-    $ pip install -r requirements_test.txt
+    $ python -m  pip install -r requirements_test.txt
```

Comment:
did `pip install -r requirements_test.txt` not work for you?

the standard way to run `pip` is to call it directly

Reply:
Issue with that was that the machine was attempting to use default python version on the machine 2.7 even after having an active environment so failed for me. Happy to revert if you think former is more standard